src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -557,6 +575,17 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 			// Reset the completion item as the object reference must remain the same but its
 			// contents will differ across syncs. This is done so we don't need to reassign the
 			// model and the slowdown/flickering that could potentially cause.
+			const completionItems = this._addPropertiesToInlineCompletionItem(completions);
+			if (lineContext && completionItems) {
+				this._model = new TerminalCompletionModel(
+					[
+						...completionItems,
+						this._inlineCompletionItem,
+					],
+					lineContext
+				);
+				this._showCompletions(this._model, false);
+			}
```

Comment:
I don't think you should need this? Was it not updating documentation or something without it?